pipeline/00-ingest.R

```diff
@@ -307,8 +337,15 @@ training_data_clean <- training_data_w_hie %>%
     time_sale_day_of_week = as.integer(wday(meta_sale_date)),
     time_sale_post_covid = meta_sale_date >= make_date(2020, 3, 15)
   ) %>%
-  select(-time_interval) %>%
+  select(-any_of(c("time_interval"))) %>%
   relocate(starts_with("sv_"), .after = everything()) %>%
+  relocate("year", .after = everything()) %>%
+  relocate("meta_sale_count_past_n_years", .after = meta_sale_buyer_name) %>%
+  filter(between(
+    meta_sale_date,
+    make_date(params$input$min_sale_year, 1, 1),
+    make_date(params$input$max_sale_year, 12, 31)
+  )) %>%
```

Comment:
Need an additional filter to keep the sales within the `params$input` specified window.